### ISSUE TITLE
[FEA] support empty "to_replace" argument for cudf_series.replace

### ISSUE THREAD

**Issue by ChristofHenkel** [rapidsai/cudf#4601]
cudf_series.replace should support an empty to_replace argument like pandas does

```
vals = np.array([]).astype(int)

pd_series = pd.Series([1,1])
pd_series.replace(vals,-1)


cudf_series = cudf.from_pandas(pd_series)
cudf_series.replace(vals,-1)

  File "cudf/_libxx/replace.pyx", line 44, in cudf._libxx.replace.replace
RuntimeError: cuDF failure at: /conda/conda-bld/libcudf_1584590065083/work/cpp/src/replace/replace.cu:487: values_to_replace and replacement_values size mismatch.
```

**Comment by galipremsagar** [comment]
It is good to have an issue for it, just FYI: This is being handled in #4543.

**Comment by galipremsagar** [comment]
Fixed by: #4543

**Comment by galipremsagar** [comment]
This doesn't seem to be fixed, slipped through somehow. I have a fix for this issue, will be up soon after testing.
